chapter06/drjt/ex1.pl

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/perl
+use 5.010;
+use warnings;
+use strict;
+use Storable qw( nstore retrieve );
+
+my $storage = 'coconet.archive';
+
+my %total_bytes;
+if ( -r $storage ) {
+  %total_bytes = %{ retrieve( $storage ) };
+}
```

Comment:
Mostly there it's just me working from the chapter05 answer. That said, I would also have to dereference every other call to `total_bytes`, so it seems like more hassle than gain